variables.tf

```diff
@@ -13,6 +13,13 @@ variable "expel_aws_account_arn" {
   default     = "arn:aws:iam::012205512454:user/ExpelCloudService"
 }
 
+# expel_assume_role
```

Comment:
Nit (non-blocking): the rest of the comments end with a colon `:`, so this should for consistency's sake.